syntax = "proto3";

import "PB_Color.proto";

// All message type names are prefixed with "PB_" to avoid collisions
// C++ classes already defined in code and Protobuf generated types.

message PB_ReqPing {
    PB_Color color = 1;
}
